test/integration/Highlight.e2e.test.js

```diff
@@ -0,0 +1,32 @@
+// <reference types="Cypress" />
+describe('Highlight', () => {
+    beforeEach(() => {
+        cy.visit('/');
+    });
+
+    it('should create a new highlight on a document', () => {
+        // Show the preview
+        cy.showPreview(Cypress.env('FILE_ID_DOC'));
+
+        // Wait for the empty highlight layer to be present
+        cy.getByTestId('ba-Layer--highlight');
+
+        cy.document().then(document => {
+            // Record the number of annotations
+            const { length } = document.querySelectorAll('[data-testid^="ba-AnnotationTarget"]');
+
+            // Enter highlight creation mode
+            cy.getByTestId('bp-AnnotationsControls-highlightBtn').click();
+
+            // Add a highlight annotation on the document
+            cy.selectText();
+            cy.submitReply();
+
+            // Assert that one more annotation is created on the document
+            cy.get('[data-testid^="ba-AnnotationTarget"]').should('have.length', length + 1);
```

Comment:
Can we select just the highlight annotations? Seems like that would simplify things quite a bit and also avoid having to destroy/recreate the test files for each test and/or suite, which will be complicated.

Reply:
Better idea 🥇